src/etc/inc/interfaces.inc

```diff
@@ -3035,11 +3035,21 @@ function kill_dhcp6client_process($interface) {
  	}
 
  	$i = 0;
+ 	
+ 	$pid = find_dhcp6c_process($interface);
+	
+	if($pid > 0){
+		mwexec("/usr/bin/logger -t mwtag 'found dhcp6 process {$pid}'"); // debug info only
+	}
+	else {
+		mwexec("/usr/bin/logger -t mwtag 'no dhcp6c process to kill'"); // debug info only
+		unset ($i);
+		return;
+	}
+	
  	while ((($pid = find_dhcp6c_process($interface)) != 0) && ($i < 3)) {
- 		/* 3rd time make it die for sure */
- 		$sig = ($i == 2 ? SIGKILL : SIGTERM);
- 		posix_kill($pid, $sig);
- 		sleep(1);
+ 		mwexec("kill -9 $pid");
```

Comment:
OK, tried this several times now and this does not kill the dhcp6c process when a release is done on the WAN interface.

 while ((($pid = find_dhcp6c_process($interface)) != 0) && ($i < 3)) {
 		/* 3rd time make it die for sure */
 		$sig = ($i == 2 ? SIGKILL : SIGTERM);
 		posix_kill($pid, $sig);
 		sleep(1);
	}

This does, every time.
mwexec("kill -9 $pid");
sleep(1);

Anyone else like to try it?